Marlin/language_en.h

```diff
@@ -445,12 +445,20 @@
 #ifndef MSG_ZPROBE_OUT
   #define MSG_ZPROBE_OUT                      "Z probe out. bed"
 #endif
+/* unused
 #ifndef MSG_YX_UNHOMED
   #define MSG_YX_UNHOMED                      "Home X/Y before Z"
 #endif
 #ifndef MSG_XYZ_UNHOMED
   #define MSG_XYZ_UNHOMED                     "Home XYZ first"
 #endif
+*/
+#ifndef MSG_HOME
+  #define MSG_HOME                            "Home"  // Used as MSG_HOME " " MSG_X  MSG_Y  MSG_Z " " MSG_FIRST
```

Comment:
I'm not sure how this will work with other languages. Can "first" always come last?